js/components/portal-dashboard/level-viewer.tsx

```diff
@@ -15,18 +15,23 @@ interface IProps {
   currentQuestion?: Map<string, any>;
   toggleCurrentActivity: (activityId: string) => void;
   toggleCurrentQuestion: (questionId: string) => void;
+  leftPosition: number;
 }
 
 export class LevelViewer extends React.PureComponent<IProps> {
   render() {
-    const { activities } = this.props;
+    const { activities, leftPosition } = this.props;
+    const position = {
+      left: leftPosition,
+    };
     return (
       <div className={css.levelViewer} data-cy="level-viewer">
-        <div className={css.activityButtons}>
+        <div className={css.activityButtons} style={position}>
           {
             activities.toArray().map(this.renderActivityButton(activities.toArray().length))
           }
         </div>
+        <div className={css.cover} />
```

Comment:
This div is simply here so that the animations continue to look nice (covers the content as it slides in from the bottom).